go.mod

```diff
@@ -30,14 +30,14 @@ require (
 	github.com/hashicorp/go-retryablehttp v0.5.4
 	github.com/hashicorp/vault/api v1.0.4
 	github.com/justinas/alice v1.2.0
+	github.com/lib/pq v1.8.0
 	github.com/magefile/mage v1.10.0 // indirect
 	github.com/oxtoacart/bpool v0.0.0-20190530202638-03653db5a59c
 	github.com/sirupsen/logrus v1.8.1
 	github.com/spf13/cobra v1.1.3
 	github.com/spf13/pflag v1.0.5
 	github.com/spf13/viper v1.7.1
 	github.com/stretchr/testify v1.7.0
-	github.com/swaggo/swag v1.7.1
```

Comment:
We keep going back and forth with this? this you try to run `$> make install-swag` already?

Reply:
Is it okay now? I ran the command and then `go mod tidy`